index.html

```diff
@@ -4189,7 +4189,22 @@ <h3>JSON-LD</h3>
         </p>
 
         <p>
-[[!JSON-LD11]] is useful when extending the data model described in this
+Linked Data is modeled using Resource Description Framework (RDF).
+RDF [[?RDF11-CONCEPTS]] is a technology for modeling graphs of statements. Each statement is a
+single <i>subject&#x2192;property&#x2192;value</i> (also known as
+<i>entity→attribute→value</i>) relationship, which is referred to
+as a <a>claim</a> in this specification. JSON-LD [[?JSON-LD11]] is a technology that enables
+the expression of RDF using idiomatic JSON, enabling developers familiar with
+JSON to write applications that consume RDF as JSON. In general, subjects are
```

Comment:
This sentence seems to repeat what is said in the first paragraph...

(I actually begin to wonder whether the first paragraph should not be removed altogether.)

Reply:
I do feel this new "second paragraph" is much clearer about what Linked Data is, what it provides, and how JSON-LD is an implementation of that which empower extensibility. I'm in favor of removing the first paragraph in favor of this new second one.

@msporny @dlongley any objections?

Reply:
Yes, that should work. Let's try that and see how it reads.

Reply:
Merged the two--kept the first sentence of the first paragraph and removed the rest of the first paragraph:
[409bfcd](https://github.com/w3c/vc-data-model/pull/1413/commits/409bfcdd59c918c9bc2e9cdc142242aef1c0234e)